docs/articles/documentation/extending-testcafe/custom-browser-provider-plugin/index.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: docs
+title: Custom Browser Provider Plugin
+permalink: /documentation/extending-testcafe/custom-browser-provider-plugin/
+---
+# Custom Browser Provider Plugin
+
+TestCafe has built-in support for testing in popular browsers (see [Using Browsers](../../using-testcafe/common-concepts/using-browsers.md)).
+If you need to use a specific browser (for example, a browser from a cloud service), you can create a **custom browser provider plugin** that will allow to perform different actions on the browser windows during testing:
+to open and close it, to take screenshots of the browser window, etc.
```

Comment:
Let's not use `specific` when we mean something closer to `special`. Because we more often use `specific` meaning `particular`.
(btw, `special` also doesn't fit here well)

I wouldn't say `can` here - the only way to connect to an otherwise unsupported browser is to create a browser provider plugin

Also, I would place a period after the word `plugin`. We need it to use the browser. That's it. Not opening, closing or screenshoting - just to test against it. Everything else is secondary and distracts the reader at this point.